.gitignore

```diff
@@ -0,0 +1,5 @@
+/.project
+build/
```

Comment:
To be consistent with the other directories and openj9-openjdk-jdk9, `build/` should start with a slash